styles.css

```diff
@@ -268,3 +277,225 @@ textarea {
 }
 
 /* footer end */
+
+/* Desktop Version Start */
+
+@media (min-width: 1440px) {
+  /* desktop Nav */
```

Comment:
You have done great at adding the media query to display the desktop version of the website. For your design, using the min-width as 1440px means that the mobile version remains until that size. For example, a 1366px displays as shown below:

![image](https://user-images.githubusercontent.com/20554583/199486405-41724040-990b-4d91-b9c4-de85a540001d.png)

This is more because you have given the mobile view a fixed with of 375px. However, the mobile version is expected to be responsive until the desktop screen size.

Kindly consider making adjustments to this.